tests/sqllogictests/suites/mode/standalone/ee/explain.test

```diff
@@ -342,38 +342,30 @@ insert into t1 values(1, parse_json('{"a":[1,2,3],"b":{"c":10}}'))
 query T
 explain select a, v['b'] from t1
 ----
-EvalScalar
-├── output columns: [t1.a (#0), v['b'] (#2)]
-├── expressions: [v:b (#3)]
-├── estimated rows: 1.00
-└── TableScan
-    ├── table: default.test_virtual_db.t1
-    ├── output columns: [a (#0), v:b (#3)]
-    ├── read rows: 1
-    ├── read bytes: 137
-    ├── partitions total: 1
-    ├── partitions scanned: 1
-    ├── pruning stats: [segments: <range pruning: 1 to 1>, blocks: <range pruning: 1 to 1, bloom pruning: 0 to 0>]
-    ├── push downs: [filters: [], limit: NONE, virtual_columns: [v:b]]
-    └── estimated rows: 1.00
+TableScan
```

Comment:
Please rename the `ee/explain.test` file to `ee/virtual_column.test`(or spill out the virtual column test), don't mix them in one, it's hard to maintain.